catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/email/EmailNotifier.java

```diff
@@ -167,34 +148,21 @@ private void sendEmailForWorkspace(WorkspaceMetacardImpl workspaceMetacard, Long
 
             mimeMessage.setText(emailBody);
 
-            LOGGER.debug("Attempting to send email to mailHost : {}", mailHost);
+            LOGGER.trace("Attempting to send email");
 
-            Transport.send(mimeMessage);
+            smtpClient.send(mimeMessage);
```

Comment:
An audit should be written anytime an email message is sent. It should include who the recipient is.